testing/e2e/workload_test.go

```diff
@@ -393,26 +599,19 @@ func TestCreateFromGitWithAnnotations(t *testing.T) {
 			Command: *it.NewTanzuAppsCommandLine(
 				"workload", "get", "test-create-git-annotations-workload", namespaceFlag),
 			Prepare: func(ctx context.Context, t *testing.T) (context.Context, error) {
-				r, w, err := pty.Open()
-				if err != nil {
+				var err error
+				if ctx, err = createPtyTerminal(ctx); err != nil {
 					t.Fatalf("error while opening pty %v", err)
 				}
-				originalStdout := os.Stdout
-				os.Stdout = w
-
-				ctx = context.WithValue(ctx, "reader", r)
-				ctx = context.WithValue(ctx, "writer", w)
-				ctx = context.WithValue(ctx, "stdout", originalStdout)
 
 				return ctx, nil
 			},
 			Verify: func(ctx context.Context, t *testing.T, output string, err error) {
 				if regexPod.FindString(output) == "" {
 					t.Fatalf("expected Pod results in output %v", output)
 				}
-				decodedString := strconv.QuoteToASCII(output)
-				if regexEmoji.FindString(decodedString) == "" {
-					t.Fatalf("output should have Emoji")
+				if !emojisExistInOutput(output) {
+					t.Fatalf("get output should have Emoji")
```

Comment:
I think the output, since it might be such a big string, can create some noise. That's why I didn't add it in the first place.. But I can sure append it to the error message